src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -24,7 +26,6 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "ComputerInfo",
         HelpUri = "https://go.microsoft.com/fwlink/?LinkId=799466")]
-    [Alias("gin")]
```

Comment:
There are other benefits to the Alias attribute:
* AllScope is terrible for performance - there is little benefit to have most of these aliases as AllScope
* Consistency - if a non-default runspace is constructed and imports one of these modules, that runspace would not get the aliases defined by default. For example:
```powershell

function DoIt($createArg)
{
    [powershell]::Create($createArg).
        AddScript('$ExecutionContext.InvokeCommand.GetCommand("clc", "Alias")').
        Invoke()
}

DoIt ([System.Management.Automation.RunspaceMode]::NewRunspace)
DoIt ([System.Management.Automation.RunspaceMode]::CurrentRunspace)

$iss = [InitialSessionState]::Create()
$iss.ImportPSModule('Microsoft.PowerShell.Management')
$iss.LanguageMode = 'FullLanguage'

DoIt $iss
```

This prints:
```
CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Alias           clc -> Clear-Content
Alias           clc -> Clear-Content
```

Note - only 2 results instead of 3. This is a little weird because Clear-Content is in `Microsoft.PowerShell.Management`, so why isn't `clc`?

We should perhaps consider adding support for AllScope/ReadOnly in the Alias attribute, perhaps as an internal only option so that it isn't abused.